cryptoutil/digestset.go

```diff
@@ -114,9 +114,9 @@ func (first DigestSet) Equal(second DigestSet) bool {
 	return hasMatchingDigest
 }
 
-func (ds DigestSet) ToNameMap() (map[string]string, error) {
+func (first DigestSet) ToNameMap() (map[string]string, error) {
```

Comment:
Can you change this to *DigestSet? I still don't have a lot of context on this code but, why the name change to `first`?

Reply:
👍